circus-train-housekeeping/src/test/java/com/hotels/bdp/circustrain/housekeeping/LegacyReplicaPathRepositoryTest.java

```diff
@@ -0,0 +1,118 @@
+package com.hotels.bdp.circustrain.housekeeping; /**
```

Comment:
why is `package` before the file header?

Reply:
+1